universe/wrappers/__init__.py

```diff
@@ -15,6 +15,7 @@
 from universe.wrappers.vectorize import Vectorize, Unvectorize, WeakUnvectorize
 from universe.wrappers.vision import Vision
 from universe.wrappers.recording import Recording
+from universe.wrappers.monitoring import Monitored
```

Comment:
Ok, I see the architecture you're going for. Why have the separate `monitor` sub-object? I think it's fine to just stick all the functionality onto the wrapper directly.